SU-SWS/stanford_basic
# D8CORE-1260: Table Style Revisions

# READY FOR REVIEW

# Summary
- Revised table styles
- Updated to Decanter 6.0.3
- Fixed a few smaller compile things with aliases and paths
- Added the dist directory to codeclimate's ignore.

# Review By (Date)
- Feb 12th

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Clear all caches
3. Create a new page with a wysiwyg paragraph
4. Create several combinations of tables using the WYSIWYG
5. Validate the new look and feel against the mock in the ticket
https://stanfordits.atlassian.net/secure/attachment/40241/40241_desired-table-theming.png
6. Review non-related-to-the-table-styles changes and ask questions about anything that looks odd.

# Affected Projects or Products
- D8CORE-1260

# Associated Issues and/or People
- D8CORE-1260
- FYI @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
